Kristina_Beresneva_4_1.py

```diff
@@ -0,0 +1,40 @@
+#Author Kristina Beresneva
+#25.02.2022
+#task: создать калькулятор
+def plus(a, b):                       #определяем функции математических операций
+    print ("Сумма: ", a, "+", b, "=", a + b)
```

Comment:
```suggestion
       return first+second
```
These changes can be applied to all functions